enumfields/fields.py

```diff
@@ -34,14 +34,29 @@ def __set__(self, obj, value):
         obj.__dict__[self.field.name] = python_value
 
 
+class CastOnSaveDescriptor:
```

Comment:
teoreticky by šlo přepast na fieldě pre_save a v ní přenastavit initial_field_name. Volá se až v sql při save, ale nezkoušel jsem to

Reply:
btw také by šlo navázat post save signál na model a tam to přepsat, bylo by to mnohem lepší

Reply:
jen aby ses nenapálil weak musí být v takovém případě nastaven na True https://docs.djangoproject.com/en/2.2/topics/signals/#listening-to-signals

Reply:
this has been removed